documentation-site/static/examples/dnd-list/varyingHeights.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {StatefulList} from 'baseui/dnd-list';
+
+export default () => (
+  <StatefulList
+    initialState={{
+      items: [
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+        'Donec ornare.',
+        'Morbi malesuada id ante ac tincidunt. Phasellus at varius enim, fringilla pretium lorem. Integer placerat, est in aliquam tempus, ex urna hendrerit quam, eu sagittis nulla lorem eu magna.',
+        'Morbi nibh nunc.',
+        'Nunc consequat erat id ante mollis tincidunt in a nulla.',
+      ],
+    }}
+    onChange={console.log} // eslint-disable-line no-console
```

Comment:
i think eslint is ignored in these files. can the comment be removed so that site visitors can see the relevant information?